backend/src/main/java/org/polsl/backend/controller/SoftwareController.java

```diff
@@ -66,6 +59,7 @@ public SoftwareController(SoftwareService softwareService) {
    * @param request struktura {@link SoftwareDTO} zawierająca dane nowego oprogramwoania
    * @return informacja o poprawnym utworzeniu oprogramowaniu
    */
+  @CrossOrigin(origins = "http://localhost:3000")
```

Comment:
Jaki miałeś problem, że dodałeś tą linijkę? Nie sądzę, żebyśmy powinni za każdym razem pisać tą linijkę

Reply:
Serwer nie zwracał mi odpowiedzi przez Cross-Origin Resource Policy, z tego co zrozumiałem to serwer nie zna mojego hosta i się nie podzieli odpowiedzią. Tymczasowe rozwiązanie, być może da się to przypisać do całego kontrolera albo zrobić lepiej, nie mam pojęcia aktualnie.